pkg/apis/core/v1beta1/types_seed.go

```diff
@@ -362,6 +362,8 @@ const (
 	SeedExtensionsReady ConditionType = "ExtensionsReady"
 	// SeedGardenletReady is a constant for a condition type indicating that the Gardenlet is ready.
 	SeedGardenletReady ConditionType = "GardenletReady"
+	// SeedSystemComponentsHealthy is a constant for a condition type indicating the system components health.
+	SeedSystemComponentsHealthy ConditionType = "SystemComponentsHealthy"
```

Comment:
This will need some extra logic in https://github.com/gardener/gardener/blob/master/pkg/controllermanager/controller/shoot/shoot_conditions_control.go because both shoots and seeds will have the condition `"SystemComponentsHealthy"` and the controller won't be able to merge those correctly: https://github.com/gardener/gardener/blob/179cf336ce5cfaa8dc3b0ef08ea1aba87ee0765c/pkg/controllermanager/controller/shoot/shoot_conditions_control.go#L82

Reply:
@timuthy wdyt about renaming the one for the seed and call it `SeedSystemComponentsHealthy`?

Reply:
I've added merging function and tests for it.